blocks/related-blogs/related-blogs.js

```diff
@@ -6,15 +6,15 @@ const maxArticlesToShow = 3;
 
 function buildHeadline(parent, tagConf) {
   const head4 = document.createElement('h4');
-  const text4head = document.createTextNode(`More ${tagConf}`);
+  const text4head = document.createTextNode(`${window.placeholders?.default?.more || 'More'} ${tagConf}`);
   head4.appendChild(text4head);
   parent.appendChild(head4);
 }
 
 function buildCTASection(parent) {
   const buttonRow = document.createElement('div');
   buttonRow.classList.add('related-button-row');
-  buttonRow.innerHTML = '<a href=\'/insights\' class="button secondary">Blog Overview</a>';
+  buttonRow.innerHTML = `<a href=\\'/insights\\' class="button secondary">${window.placeholders?.default?.blogOverview || 'Blog Overview'}</a>`;
```

Comment:
I would assume you could remove the backslashes completely

Reply:
done